ci/Jenkinsfile

```diff
@@ -0,0 +1,48 @@
+node {
+    cleanWs()
+    dir('src') { git 'https://github.com/managarm/bootstrap-managarm.git' }
+    docker.build('managarm_cienv', 'src/ci/docker/').inside {
+        dir('build') {
+            stage('Build system') {
+                sh '''#!/bin/sh
+                set -xe
+                xbstrap init ../src || true
```

Comment:
quick remark about this line: `|| true` is unnecessary if `cleanWs()` is used. I used it to speed up prototyping. It's not mandatory, but it should have little to no effect as the only effect `init` has is a symlink